src/App.tsx

```diff
@@ -1,11 +1,31 @@
-import Navbar from "./common/component/Navbar"
-
+import { Routes, Route, BrowserRouter as Router } from "react-router-dom";
+import { ThemeProvider } from '@mui/material';
+import theme from './theme/theme'; 
+import Layout from "./common/component/layout/Layout";
+import Home from "./pages/home/Home";
+import Bracelates from "./pages/bracelets/Bracelates";
```

Comment:
remove unwanted Import

Reply:
it must use in app.tsx.no un wanted import